mobile/src/libs/components/input-search/input-search.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { type Dispatch, type SetStateAction } from 'react';
+
+import { TextInput } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+import { debounce as debouncedSetSearchQuery } from '#libs/helpers/helpers';
+import { useAppForm, useFormController } from '#libs/hooks/hooks';
+
+import { DEFAULT_SEARCH_PAYLOAD } from './libs/constants/constants';
+import { styles } from './styles';
+
+type Properties = {
+  placeholder: string;
+  setSearchQuery: Dispatch<SetStateAction<string>>;
+};
+
+const InputSearch: React.FC<Properties> = ({ placeholder, setSearchQuery }) => {
+  const { control } = useAppForm({
+    defaultValues: { search: DEFAULT_SEARCH_PAYLOAD.search },
```

Comment:
```suggestion
    defaultValues: DEFAULT_SEARCH_PAYLOAD,
```

Reply:
Can I declare a constant like this `const DEFAULT_SEARCH_PAYLOAD = '';` ?
If I declare it like this:  `const DEFSULT_SEARCH_PAYLOAD = { search: '',}; ` Input does not accept it because it wants a string, not an object.

Reply:
No, I understood about name. I am asking about DEFAULT_SEARCH_PAYLOAD. Can I declare it as a string : `const DEFAULT_SEARCH_PAYLOAD = '';` or it is important to declare it as an object: `const DEFSULT_SEARCH_PAYLOAD = { search: '',};`

Reply:
i didn't really get what is the problem which declaring it as object? you need to use it only as default props to pass in useAppForm

Reply:
Sorry,  I did not get the sintax right. Now I understood.